src/index.html

```diff
@@ -1,17 +1,190 @@
+          Full date of birth:
+          <input
+            type="date"
+            name="date of birth"
+            autocomplete="off"
+            required
+          >
+        </label>
+
+        <label for="" class="enter_field_terms">
+          <label for="terms">
+            I accept the term of the agreement
+          </label>
+          <input
+            type="checkbox"
+            name="accept-the-term"
+            id="terms"
+            required>
+        </label>
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend class="legend">Registration</legend>
+
+        <label for="" class="form-field">
+          E-mail:
+          <input
+            type="email"
+            name="email"
+            placeholder="email@example.com"
+            autocomplete="off"
+            required>
+        </label>
+
+        <label for="" class="enter_field_terms">
+          Password:
+          <input
+            type="password"
+            name="password"
+            minlength="5"
+            maxlength="15"
+            autocomplete="off"
+            required>
+        </label>
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend class="legend">An interesting fact about you!</legend>
+
+        <div class="enter_field">
+          Do you love cats?
+
+          <label for="">
+            <input
+              type="radio"
+              name="cats"
+              value="yes"
+            >
+            Yes
+          </label>
+
+          <label for="">
+            <input
+              type="radio"
+              name="cats"
+              value="no"
+            >
+            No
+          </label>
+        </div>
+
+        <div class="enter_field">
+          What is your favorite color?
+          <input
+            type="color"
+            id="color"
+            value="black"
+          >
+          <label for="color"></label>
+        </div>
+
+        <div class="form-field">
+          What time do you go to bed?
+          <input
+            type="time"
+            id="go_to_bed"
+            name="start-sleep"
+          >
+          <label for="go_to_bed"></label>
+        </div>
+
+        <label for="" class="enter_field">
+          What are your favorite brand of cars?
+          <select name="brand" multiple>
+            <option value="BMW">BMW</option>
+            <option value="Audi">Audi</option>
+            <option value="Lada">Lada</option>
+          </select>
+        </label>
+
+        <div class="enter_field_term">
+          How do you rate our work?
+          <input
+            type="range"
+            id="rate"
+            name="rate"
+            value="0"
+            min="0"
+            max="100"
+          >
+          <label for="rate"></label>
+        </div>
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend class="legend">Additional info:</legend>
+
+        <label for="" class="form-field">
```

Comment:
```suggestion
        <label for="" class="form-field">
```
You don't have such class in your css